packages/feedback/src/FeedbackForm.js

```diff
@@ -33,7 +32,7 @@ const FeedbackForm = ({
   onClose,
   faceOptions,
   aboutOptions,
-  aboutPlaceholder,
+  aboutLabel,
```

Comment:
If we remove `aboutPlaceholder` and replace it with `aboutLabel`, this will be a breaking change for anyone updating their version of the component. Can we keep `aboutPlaceholder` and add `aboutLabel` as a hidden label? To keep the look of feedback the same, but still hit accessibility requirements? https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text

Or show both label and placeholder. What do you all think?

Reply:
ahh, I was confusing the aboutOptions with support and thought it was a newer add, doh. Personally I'm against having both placeholder and labels showing when they say the same thing. tbh, I feel like for accessibility it's better to move away from placeholders anyway (in which case I could always keep the prop and if there's a placeholder value assign it to the label). 
but if I'm keeping this one as a placeholder do all the other inputs need to have their placeholders returned?

Reply:
If we allow separate props for label and placeholder hopefully they wouldn't both show the same value, I agree that wouldn't be a great experience.

We shouldn't keep the `aboutPlaceholder` prop and assign it to a `label` though, that would still be a breaking change and would be misleading to devs using the prop. I think we need to decide if we just introduce the new `aboutLabel` prop and have it function alongside our `aboutPlaceholder` prop, or if we remove `aboutPlaceholder` and cut a breaking change to introduce `aboutLabel`. If we keep both, all inputs would get their placeholders returned, and then have labels added.

@jordan-a-young what do you think?

Reply:
I like the breaking change idea. It seems like that will be the most straightforward and be less maintenance in the long run.